### PR TITLE
9mm ammo types to protolathe

### PR DESCRIPTION
**Pull request by BlueHNT**
## About The Pull Request
There were 2 unused ammo types of 9mm already made, so I scrambled them together and introduced them to proto lathe together with the 2 existing types.

in total there are 8 new designs
4 for all the individual Glock magazines prefilled with ammo
and 4 for all the individual ammo boxes

## Why It's Good For The Game
Glock is good, it's the only one we got.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

I ain't got nothing chief

</details>

## Changelog
:cl:
add: Added Glock AP magazine and ammo box
add: Added Glock I magazine and ammo box
add: Added Glock-related designs to proto-lathe
/:cl:

### CODE REVIEW

File: nsv13/code/modules/jobs/security/weapons.dm [hunk 1]
```diff
@@ -131,10 +203,38 @@
 	caliber = "9mm"
 	max_ammo = 15
 
+/obj/item/ammo_box/magazine/glock/ap
+	name = "pistol magazine (9mm AP)"
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/jobs/security/weapons.dm:207]:
```suggestion
	name = "pistol magazine (9mm Armour Piercing)"
```

File: nsv13/code/modules/jobs/security/weapons.dm [hunk 2]
```diff
@@ -131,10 +203,38 @@
...
+	ammo_type = /obj/item/ammo_casing/c9mm/inc
+	caliber = "9mm"
+	max_ammo = 15
+
 /obj/item/ammo_box/magazine/glock/update_icon()
     ..()
     icon_state = "[initial(icon_state)][ammo_count() ? "" : "-0"]"
 
+/obj/item/ammo_box/c9mm/ap // for colouring
+	name = "ammo box (9mm, armour piercing)"
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/jobs/security/weapons.dm:227]:
```suggestion
	name = "ammo box (9mm Armour Piercing)"
```

File: nsv13/code/modules/jobs/security/weapons.dm [hunk 3]
```diff
@@ -131,10 +203,38 @@
...
+	icon_state = "45box"
+	ammo_type = /obj/item/ammo_casing/c9mm/ap
+	max_ammo = 30
+
+/obj/item/ammo_box/c9mm/inc
+	name = "ammo box (9mm, incendiary)"
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/jobs/security/weapons.dm:233]:
```suggestion
	name = "ammo box (9mm Incendiary)"
```

File: nsv13/code/modules/jobs/security/weapons.dm [hunk 4]
```diff
@@ -107,6 +125,60 @@
...
+	name = "9mm ammobox (rubber)"
+	id = "glock_rubberammo"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/iron = 25000)
+	build_path = /obj/item/ammo_box/c9mm/rubber
+	category = list("Ammo")
+	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/glockmag
+	name = "Glock Magazine (rubber)"
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/jobs/security/weapons.dm:138]:
```suggestion
	name = "Glock Magazine (Rubber)"
```
The extra properties are also capitalized, same for the rest of the file

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]
I don't think they should be able to print the ammo boxes (in the protolathe). "Normal" ammo boxes are only acquired from a hacked autolathe and printing magazines in sec is good enough.

**Comment by BlueHNT** [comment]
> I don't think they should be able to print the ammo boxes (in the protolathe). "Normal" ammo boxes are only acquired from a hacked autolathe and printing magazines in sec is good enough.

You can just log into the autolathe to print the lethal ammo box, so it makes no actual difference. You would just have to go to cargo is the only real difference, but if you still believe so I can remove it.

(You can also build the autolathe in sec very easily omitting the walking to cargo)
1. Use ID on autolathe with security access
2. Deactivate security measures
3. Print "hacked" items.

File: nsv13/code/modules/jobs/security/weapons.dm (hunk 4 above)
Comment (Bokkiewokkie) [reply nsv13/code/modules/jobs/security/weapons.dm]: @BlueHNT I still see other designs with uncapitalized names

Comment (BlueHNT) [reply nsv13/code/modules/jobs/security/weapons.dm]: > @BlueHNT I still see other designs with uncapitalized names

Naming scheme

magazines
design Glock Magazine (Type)
obj pistol magazine (Type)

ammo boxes
design 9mm Ammo Box (Type)
obj ammo box (Type)

**Review by Bokkiewokkie — DISMISSED** [review]

File: nsv13/code/modules/jobs/security/weapons.dm [hunk 5]
```diff
@@ -110,6 +128,59 @@
 	build_path = /obj/item/ammo_casing/c9mm/rubber
 	category = list("initial", "Security")
 
+/datum/design/rubber9mm
+	name = "9mm Ammo Box (Rubber)"
+	id = "glock_rubberammo"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/iron = 25000)
+	build_path = /obj/item/ammo_box/c9mm/rubber
+	category = list("Ammo")
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/jobs/security/weapons.dm:137]:
```suggestion
	category = list("Ammo")
	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
```
You forgot this one

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

**Review by Bokkiewokkie — APPROVED** [review]

**Pull request merged**